### ISSUE TITLE
Change request process

### ISSUE THREAD

**Issue by shawnthompson** [gc-da11yn/project-management#3]
Would like to come up with a process for requesting a change on site. 

Using this issue as an example.

https://github.com/gc-da11yn/gc-da11yn.github.io/issues/564

This request

1. came in as an email, 
2. we created an issue in the main repo for the change,
3. Assigned to Julie
4. Review by AAACT
5. Discussions started in issue comments
6. Decision made in issue comments

My suggestions for improvements to this request due to the need of a discussion before the change was to be made, maybe it should have been a discussion and then when a decision is made someone would convert the discussion to an issue give clear instruction on the change that is to be made. Then someone can easily create a pull request to make the change on the site that was decided by the group. 

Issues should be kept clear and precise so they can be actioned quickly and pull requests can be made. 

GitHub allows us to convert open issues to discussions easily. 

We could even use the voting system to get the community to up vote discussions they might think need a little push.

**Comment by lynelal** [comment]
'DAT content organization' and 'DAT change request process' are separate projects / issues.  They each require it's own project name.

**Comment by Syracopoulos** [comment]
Maybe we need some more specific intake options and guidance to allow people to open a discussion before the issues ticket is opened.  I'm also thinking that not everything needs to be debated. Some of the comments may already be covered by other GC guidance such as the style guide or WCAG. There also needs to be a constant triage so that all items are monitored and moving.

**Comment by lynelal** [comment]
@Syracopoulos Maybe, but we can't assume a person entering info in our tracker will understand our processes.  What one person thinks is a single issue can be multiple issues depending on internal processes. Modifying out intake form will only get us so far.

**Comment by Syracopoulos** [comment]
Agreed, we could provide guidance on when to use which though. If it is something "open for discussion", then people can engage in that conversation. If it is something broken or goes against a standard, then that is a direct issue and pull request. Having a dedicated triage person to demystify tickets would help as well obviously
